### PR TITLE
Merge `app-branch` to `development`

### PR DESCRIPTION
**Pull request by o-rashad**
New Skills learned 📋 :
- Develop a mental tree steps for designing and manipulating projects evolving DOM elements.
- Hands-on DOM elements Js best coding practice.
- Practicing ES6 Functional programming and OOP concepts.

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  title.setAttribute('type', 'text');
+  title.setAttribute('placeholder', 'Title');
+
+  des.setAttribute('type', 'text');
+  des.setAttribute('placeholder', 'Description');
+
+  dueDate.setAttribute('type', 'date');
+  dueDate.setAttribute('placeholder', 'Due Date');
+
+  priority.setAttribute('type', 'number');
```
Comment (bellom) [inline src/common.js]: Opps, your priority should not be a number.

It should be either of the following; low, medium, or high

image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @od-c0d3r  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+## To get a local copy of this repository kindly follow the steps below.
+- Scroll to top of this current repository
+- Click on the `Code` button with background color green on the right end corner
+- Click on the clipboard icon on the extreme right of the dropdown to copy the repository link
+- In your local PC, open your terminal or command prompt in the folder you would like to clone this repository into
+- Type `git clone (copied link)` on the currently opened terminal or command prompt
+
+# Getting Started 
+
+- Open your Terminal and Navigate to the project destination and type `npm install`
```
Comment (bellom) [inline README.md:25]: Kindly complete the steps that would take a user to run your app locally.

running only `npm install` won't start the app.

Kindly write the correct steps and fix your running commands like `npm run server`

**Comment by o-rashad** [comment]
Thank you @bellom for the quick response review. ⚡💯 
Working on your changes, and gonna submit ASAP 👍🏼

File: README.md (hunk 2 above)
Comment (o-rashad) [reply README.md:25]: ✔️

File: src/common.js (hunk 1 above)
Comment (o-rashad) [reply src/common.js]: ✔️

**Review by rukundoeric — APPROVED** [review]
Hi @od-c0d3r ,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**